fleetctl/stop.go

```diff
@@ -85,6 +85,11 @@ func runStopUnit(args []string) (exit int) {
 	}
 
 	exit = tryWaitForUnitStates(stopping, "stop", job.JobStateLoaded, getBlockAttempts(), os.Stdout)
+	if exit == 0 {
+		stdout("Successfully stopped units %v.", stopping)
```

Comment:
Why is this to stdout and not stderr? stdout is basically part of the contract of the output (i.e. this would become API)